com_file_arch.md

```diff
@@ -0,0 +1,7 @@
+# Communication and File-Archiving Commands
+
+Computers running the Unix operating system are often connected to other computers. This ability allows transfer of files between computers, email, websites etc., File archiving is included in this section because files are often archived between different computers on a network.
```

Comment:
```suggestion
Computers running the Unix operating system are often connected to other computers. This ability allows transfer of files between computers, email, websites etc. File archiving is included in this section because files are often archived between different computers on a network.
```